src/utils/breakpoints.ts

```diff
@@ -20,20 +21,16 @@ export const bpMaxSM = `@media (max-width: ${bp.md - 1}px)`
 export const bpMaxMD = `@media (max-width: ${bp.lg - 1}px)`
 export const bpMaxLG = `@media (max-width: ${bp.xl - 1}px)`
 
-const useBreakpoints = createBreakpoint(bp)
+const getBreakpoint = createBreakpoint(bp)
 
-const useBreakpoint = () => {
-  const breakpoint = useBreakpoints()
-
-  const xs = breakpoint == 'xs'
-  const sm = breakpoint == 'sm' || xs
-  const md = breakpoint == 'md' || sm
-  const lg = breakpoint == 'lg' || md
-  const xl = breakpoint == 'xl' || lg
-  const xl2 = breakpoint == '2xl' || xl
-  const xl3 = breakpoint == '3xl' || xl2
-
-  return {xs, sm, md, lg, xl, xl2, xl3}
+const breakpoints = () => {
+  const breakpoint = getBreakpoint()
+  const keys = Object.keys(bp)
+  const foundInd = keys.indexOf(breakpoint)
+  return keys
+    .map((key, ind) => ({['isMin' + key.toUpperCase()]: ind <= foundInd}))
+    .filter((_, ind) => ind)
```

Comment:
Idea here is to filter out `XS` value, because it's like `default` class in tailwindcss (`flex-row` vs `sm:flex-row`)